inst/@sym/horzcat.m

```diff
@@ -62,7 +62,7 @@
           'return sp.Matrix.hstack(*_proc),'
           };
 
-  varargin = sym(varargin);
+  varargin = cell_array_to_sym(varargin);
```

Comment:
Hi @mtmiller, even after adding the private functions in path, we can't call the constructor of sym with cells! Although I tried this with some barebone classdefs but the error didn't appear. Could there be anything specific to sym constructor  that the octave parser isn't even trying to call the constructor? It just gives the error "cannot convert cell to object"!